Client/app/src/main/java/at/aau/anti_mon/client/CreateGame.java

```diff
@@ -0,0 +1,11 @@
+package at.aau.anti_mon.client;
+
+public class CreateGame {
+    private String command;
```

Comment:
this String should be an enum with different commands, you can find them in the server

Reply:
you can also find the whole Commands class in the server, which has the same entities like CreateGame